### PR TITLE
{vis}[foss/2016b] graphviz 0.5.1 w/ Python 2.7.12 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**
All configs are derived from older foss and intel/2016b easyconfigs.

### CODE REVIEW

File: easybuild/easyconfigs/g/Graphviz/Graphviz-2.38.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    ('Tcl', '8.6.5'),
+    ('zlib', '1.2.8'),
+]
+
+builddependencies = [
+    ('M4', '1.4.17'),
+    ('SWIG', '3.0.10', '-Python-2.7.12'),
+]
+
+#patches = [
```
Comment (boegel) [inline easybuild/easyconfigs/g/Graphviz/Graphviz-2.38.0-foss-2016b.eb]: @hajgato please remove rather than only comment out?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2086.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ce2a8e2a937802a3373a02811198d97a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 26 out of 26 (9 easyconfigs in this PR)
node2605.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/013f89ab2f851a45bc382b4e84326caf for a full test report.

File: easybuild/easyconfigs/g/Graphviz/Graphviz-2.38.0-foss-2016b.eb (hunk 1 above)
Comment (hajgato) [inline easybuild/easyconfigs/g/Graphviz/Graphviz-2.38.0-foss-2016b.eb]: @boegel: I forgot to remove. Now its done.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**